### ISSUE TITLE
Problems with consecutive locked nucleic acids at the end of the sequence?

### ISSUE THREAD

**Issue by gwenncini** [aravind-j/rmelting#2]
Why is there a IndexOutOfBoundsException if the sequence contains consecutive locked nucleic acids at the end?

Please see the examples below:
Consecutive locked nucleic acids effect - owc11 (DNA/DNA) / example from vignette
> melting(sequence = "GALCLC", nucleic.acid.conc = 0.0001, hybridisation.type = "dnadna", Na.conc = 1, method.consecutive.locked = "owc11")
[1] 12.94323

add an locked nucleic acid at the end
> melting(sequence = "GALCLCL", nucleic.acid.conc = 0.0001, hybridisation.type = "dnadna", Na.conc = 1, method.consecutive.locked = "owc11")
[1] NA
ERROR: java.lang.IndexOutOfBoundsException: Index: 4, Size: 4

 delete locked nucleic acids in the middle
> melting(sequence = "GALCCL", nucleic.acid.conc = 0.0001, hybridisation.type = "dnadna", Na.conc = 1, method.consecutive.locked = "owc11")
[1] 9.302131

locked nucleic acids -- the last two positions
> melting(sequence = "GACLCL",  nucleic.acid.conc = 0.0001, hybridisation.type = "dnadna",  Na.conc = 1, method.consecutive.locked = "owc11")
[1] NA
ERROR: java.lang.IndexOutOfBoundsException: Index: 4, Size: 4

  locked nucleic acids -- only the last position
> melting(sequence = "GACCL",  nucleic.acid.conc = 0.0001, hybridisation.type = "dnadna",   Na.conc = 1, method.consecutive.locked = "owc11")
[1] 4.459016

**Comment by Permafacture** [comment]
It's an issue with Melting 5, which rmelting is just a wrapper for

```
melting -S GALCLCL -H dnadna -P 2e-07 -E Na=0.05 -C CTGG
SEVERE : Index 4 out of bounds for length 4
```

**Comment by aravind-j** [comment]
Yes, it seems to be an issue with [MELTING 5](https://www.ebi.ac.uk/biomodels/tools/melting/). I will add a warning in the documentation about this.
